go.mod

```diff
@@ -282,3 +282,5 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	stash.kopano.io/kgol/kcc-go/v5 v5.0.1 // indirect
 )
+
+replace github.com/cs3org/reva/v2 => github.com/kobergj/reva/v2 v2.0.0-20220919125358-e230a6883013
```

Comment:
I needed to rebase the branch, sorry. We need to update the commitID otherwise this won't build